app/models/organization.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Organization < ApplicationRecord
+  after_create :create_tenant
```

Comment:
Add the model validations for Organization model

Reply:
done